.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
- [ ] [optional] @shuja-shah great work implementing the changes requested, it would be nice for you to add a readme file to your project.